teuthology/suite/run.py

```diff
@@ -357,8 +396,14 @@ def build_base_config(self):
         job_config.timestamp = self.timestamp
         job_config.priority = self.args.priority
         job_config.seed = self.args.seed
+        if self.args.subset and self.args.descr:
+            util.schedule_fail("--subset is not compatible with --descr")
+        if self.args.suite and self.args.descr:
+            util.schedule_fail("--suite is not compatible with --descr")
         if self.args.subset:
             job_config.subset = '/'.join(str(i) for i in self.args.subset)
+        if self.args.descr:
+            job_config.suite = "rerun"
```

Comment:
instead of "rerun" maybe it is better to define somehow a suite name from the descriptions, but this is another question, another good question is it supposed to have scheduled descriptions from different suites, this is very interesting.